src/client/scripts/room.js

```diff
@@ -1,41 +1,44 @@
 "use strict";
 /* global Cards:false Rooms:false*/
 /**
-*@author THouse
 *@purpose To provide the room template with data to display
 **/
 
 Template.room.onCreated(function () {
-    this.subscribe("cards");
+    Meteor.autorun(function() {
```

Comment:
i just went through and made every subscribe only subscribe to the room they are in, there probably is some redundant subscribes, however every subscribe was subscribing to everything not just what it needed and that is what I was trying to fix.